signs.lua

```diff
@@ -701,7 +701,8 @@ streets.register_sign = function(name, def)
             if (param2 >= 0 and param2 <= 4)  and exact_point then
                 local component = (param2 == 0 or param2 == 2) and "z" or "x"
                 behind_pos[component] = behind_pos[component] + (param2 <= 1 and 1 or -1)
-                if math.abs(pos[component] - exact_point[component]) >= .75 then
+                local is_pole = string.sub(node.name, #node.name - 9, #node.name)
+                if math.abs(pos[component] - exact_point[component]) >= .75 and is_pole ~= "_polemount" then
```

Comment:
```suggestion
                if math.abs(pos[component] - exact_point[component]) >= .75 and minetest.registered_nodes[node.name .. "_polemount"] then
```

Unfortunately, I can't reproduce this problem. But wouldn't it be more elegant to check if the node is registered?